tendrl/node_agent/manager/manager.py

```diff
@@ -99,12 +99,14 @@ def __init__(self, machine_id):
                     is None:
                 utils.delete_local_node_context()
 
+        node_id = utils.get_local_node_context()
         super(
             NodeAgentManager,
             self
         ).__init__(
             "node",
-            utils.get_local_node_context(),
+            node_id,
+            node_id,
```

Comment:
Well its redundant, and integration_id is meant to be generated by the tendrl-api and it is definitely not node_id